suit_generator/suit/security.py

```diff
@@ -271,3 +318,55 @@ class SuitDelegationChain(SuitList):
     """Representation of SUIT delegation chain."""
 
     _metadata = Metadata(children=[SuitDelegation])
+
+
+### Encryption
+
+class SuitCiphertextBytes(SuitHex):
+    """Representation of SUIT ciphertext bytes."""
+
+    pass
+
+class CoseEncryptCiphertext(SuitUnion):
+    """Representation of COSE_Encrypt_ciphertext item."""
+
+    _metadata = Metadata(
+        children=[
+            SuitNull,
+            SuitCiphertextBytes,
+        ]
+    )
+
+class CoseRecipient(SuitTupleNamed):
+    """Representation of COSE_Recipient item."""
+
+    _metadata = Metadata(
+        map={
+            "protected": cbstr(SuitHeaderMapOptional),
+            "unprotected": SuitHeaderData,
+            "ciphertext": CoseEncryptCiphertext,
+            "recipients*": SuitList, # TODO: should be CoseRecipientList, but we have to solve the circular dependencies somehow...
```

Comment:
If you look for `# Fix cyclic dependencies between types` comment you'll find how this can be fixed :slightly_smiling_face:

Reply:
Ok, fixed this